client/components/mma/cancel/tierThree/TierThreeCancellationReasons.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+	inOrderToImproveSubs,
+	standardAlternateFeedbackIntro,
+} from '../cancellationConstants';
+import type { CancellationReason } from '../cancellationReason';
+import { BreakFromNewsWithAlternative } from '../GenericSaveBodyResponses';
+
+export const tierThreeCancellationReasons: CancellationReason[] = [
+	{
+		reasonId: 'mma_editorial',
```

Comment:
the doc doesn't mention what the reasonIds should be.  These all look reasonable though.